packages/shared_preferences/shared_preferences_macos/README.md

```diff
@@ -31,4 +31,4 @@ dependencies:
   ...
 ```
 
-[1]: ../shared_preferences/shared_preferences
+[1]: ../
```

Comment:
This links to a directory of folders on Github, if somebody didn't already know what it was it would be confusing. I think it would make more sense to link to the interface package on pub, WDYT?

Reply:
I don't have a strong preference, but I think it's better to be consistent and other plugins do the same. Maybe at some point we should change them all?

Reply:
Sure, I think that makes more sense.